app/event_source/publishers/families/notices/faa_totally_ineligible_notice_requested_publisher.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Publishers
+  module Families
+    module Notices
+      # This class will register event 'fre_notice_generation.requested'
```

Comment:
is this comment relevant ? or this should be `faa_totally_ineligible_notice.requested`